prime-angular-frontend/src/app/modules/adjudication/shared/components/enrollee-table/enrollee-table.component.html

```diff
@@ -163,9 +163,13 @@
               [class]="row.currentStatusCode | configCode: 'statuses' | case: 'space' : 'kebab'">
 
             <div class="d-flex align-items-center">
-              <span class="mr-1">{{ row.currentStatusCode | configCode: 'statuses' | default }}</span>
+              <span class="mr-1 status"
+                    [class.paper-enrolment-complete-incomplete]="filterPaperEnrolmentStatus(row)">
```

Comment:
Is the thought that a paper enrolment (incomplete or complete) will be Blue in colour?

If that is the goal, you should just do something like this
``` [class.paper-enrolment]="row.gpid | contains: PAPER_ENROLLEE_GPID_PREFIX" ``` 
Since we do that elsewhere, but looking at that you may want to change since technically NOBCSC can be found in another part of the gpid. So you can use startsWith instead and with a pipe or not.

Reply:
If I do that, then a declined or locked paper enrolment status will also be in blue which is not correct (not sure if a paper enrolment should be locked or declined, but it's something an admin can do) hence I used the name `paper-enrolment-complete-incomplete`  to separate it from the other coloured statuses. `filterPaperEnrolmentStatus()` already uses `startsWith` .